### PR TITLE
Implemented DTO and method for fetching DailySummaryConfiguration from backend

### PR DESCRIPTION
**Pull request by jonasofrich**

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Models/DailySummaryConfigurationDTO.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace NDB.Covid19.Models
+{
+    public class DailySummaryConfigurationDTO
+    {
+        public Xamarin.ExposureNotifications.DailySummaryConfiguration DailySummaryConfiguration { get; set; }
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/Models/DailySummaryConfigurationDTO.cs:6]: Please use static import for so long paths ;)

File: NDB.Covid19/NDB.Covid19/WebServices/ExposureNotification/ExposureNotificationWebService.cs [hunk 2]
```diff
@@ -104,7 +104,21 @@ public async Task<Xamarin.ExposureNotifications.Configuration> GetExposureConfig
             }
 
             return null;
+        }
+
+        public async Task<Xamarin.ExposureNotifications.DailySummaryConfiguration> GetDailySummaryConfiguration()
```
Comment (nc-kano) [inline NDB.Covid19/NDB.Covid19/WebServices/ExposureNotification/ExposureNotificationWebService.cs:109]: ditto

File: NDB.Covid19/NDB.Covid19/Models/DailySummaryConfigurationDTO.cs (hunk 1 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/Models/DailySummaryConfigurationDTO.cs]: Fixed

File: NDB.Covid19/NDB.Covid19/WebServices/ExposureNotification/ExposureNotificationWebService.cs (hunk 2 above)
Comment (jonasofrich) [reply NDB.Covid19/NDB.Covid19/WebServices/ExposureNotification/ExposureNotificationWebService.cs]: Fixed

**Review by nc-kano — APPROVED** [review]

**Pull request merged**